src/main/java/com/unzer/payment/models/AdditionalTransactionData.java

```diff
@@ -51,4 +61,34 @@ public AdditionalTransactionData setRiskData(RiskData riskData) {
         this.riskData = riskData;
         return this;
     }
+
+    /**
+     * URL to the merchant's Terms and Conditions Page
+     */
+    public String getTermsAndConditionsUrl() {
```

Comment:
Sometimes it is termsAndConditionUrl (property name)
Sometimes it is termsAndConditionsUrl (Getter, Setter )

I guess this it intended, right?

Reply:
Ok, nice.